yt_dlp/extractor/wimbledon.py

```diff
@@ -0,0 +1,60 @@
+from .common import InfoExtractor
+from ..utils import (
+    parse_duration,
+    traverse_obj,
+)
+
+
+class WimbledonIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?wimbledon\.com/en_GB/video/media/(?P<id>[0-9]+).html'
+    BRIGHTCOVE_URL_TEMPLATE = 'http://players.brightcove.net/%s/default_default/index.html?videoId=%s'
+
+    _TESTS = [{
+        'url': 'https://www.wimbledon.com/en_GB/video/media/6330247525112.html',
+        'info_dict': {
+            'id': '6330247525112',
+            'ext': 'mp4',
+            'timestamp': 1687972186,
+            'description': '',
+            'thumbnail': 'https://cf-images.eu-west-1.prod.boltdns.net/v1/static/3506358525001/97312363-ba01-4158-8a45-1e51e518c0ff/806b5b8a-22dc-4618-aac9-f73bd6202715/1920x1080/match/image.jpg',
```

Comment:
You can use regex for thumbnail URLs if they are too long or if they may change
```suggestion
            'thumbnail': r're:^https://[\w-]+\.prod\.boltdns\.net/[^?#]+/image\.jpg',
```

Reply:
Done, had to add a capture for periods in `[\.\w-]+`.